actions/telegram-send/exo.action.yaml

```diff
@@ -0,0 +1,20 @@
+name: telegram-send
+description: Bots telegram
+handler: telegram.php
+interpreter: php
+input:
+  type: object
+  required: [token, channel, text]
+  additionalProperties: false
+  properties:
+    Token:
+      type: string
+      description: Bots token
+    channel:
```

Comment:
The Telegram API refers to "ChatID"s not "Channels". so let's call this variable `chatId`. The description can then match the variable name.